### ISSUE TITLE
Loading Screens V2 Specification

### ISSUE THREAD

**Issue by xavi-dcl** [decentraland/unity-renderer#3632]
### Description
We want to do the refinement for the functionality described in this PRD: https://www.notion.so/decentraland/PRD-Loading-Screens-V2-24a870b751a8418091d11f2ed81e7d2f

Note: assume that [ADR-101: Move loading screen logic from kernel to renderer](https://adr.decentraland.org/adr/ADR-101) is already done beforehand.

### Acceptance Criteria
- [x] Technical specification
- [x] Task breakdown
- [x] Create user stories
- [x] Estimations

**Comment by Kinerius** [comment]
For this new functionality to be implemented properly we need:
- Fetching and parsing `scene.json` of the target scene to load to get the custom hints.
- Get the global settings (example: https://config.decentraland.org/explorer.json) to load the default hints.
- Define if we want priority hints, for special events like MVFW/MVMF
- Define if we do this in parallel with the current scene loading or if we load the hints first.
- Show the hints, prioritize the custom hints from the scene, and fill with defaults, being the total number to be defined.
- If the defined hints from the scene are more than the limit, select them at random BEFORE downloading them.
- Resizing and compressing wouldn't be necessary since we are downloading the full files anyways, we can dispose of them after the loading screen is gone ( except the default ones, those images could be embedded with Adressables, if not, we dispose of them if they are too big but only for WebGL (TBD)).
- The tips carrousel must be controllable with hotkeys and mouse input.
- Animations will look clunky since loading assets causes a lot of hiccups, so those are very low priority.
- Decide which background we use from the options described in the document, the blur one is the easiest.

**Comment by Kinerius** [comment]
# Task Breakdown

## Hint Request
- `HintRequestService` should request hints from different sources `IHintRequestSource` which can be local or remote. 
- All `IHintRequestSource` will yield multiple `IHint`, unless the request fails, they should return an empty list.
- Every `IHint` contains its texture URL, title, body, and source tag (Scene, Dcl, Event, etc)
- This `HintRequestService` is responsible for yielding the most optimal amount of hints (N) in the correct order (Scene > Event > Dcl), after that is decided, the Textures should be downloaded and created.
- The request for hints is asynchronous and should be cancellable at all times and every `IHint` should be disposed and if the textures were created, they should be destroyed.
- If every `IHintRequestSource` fails, the service should return an empty list of Hints.
- If the texture download fails for any hint, we show the hint with no texture.
- If any exception occurs inside the `HintRequestService`, the exception should be shown in the console and the service should return an empty list of Hints to avoid locking the loading process.

Every item described above should have its own Unit Test

This request should be done at the `LoadingScreenController` after the loading is triggered, we need to figure out if loading the hints in parallel is the best approach, if not, we should load the hints first and then the scene assets.

> **Estimation**: 5

## LoadingScreenView
`LoadingScreenView` should be modified based on the Loading [Screens V2 Specification](https://www.notion.so/decentraland/PRD-Loading-Screens-V2-24a870b751a8418091d11f2ed81e7d2f)

Most of the changes are graphical.

> **Estimation**: 3

## LoadingScreenTipsController
- This view should receive a list of `IHint` and show them.
- The view should contain the max amount of hints that can be displayed, and they should be set up after the list of `IHint` arrives. We could also use a pool. 
- All HintViews should initialize as disabled and hidden (no text, no image)
- If the list of hints is empty or the amount is less than the max amount, we should disable the rest of the HintViews. 
- When the loading is finished, this class should handle the disposal of the `IHint` and their textures.
- When the loading screen is triggered again, we should make sure that old Hints are not loaded or shown.
- The hints carousel goes to the next hints after a few (n) seconds.
- The hints carousel allows user input from keys (A or D) to go to the next or previous hint.
- The hints carousel allows mouse input to select a specific hint.
- When any hint changes, the next hint timer gets reset.

Every item described above should have its own Unit Test

> **Estimation**: 5

## Loading screen background

We should decide which background we want to use.

### Option 1: Lava Lights
Lava Lights, any bokeh shader will do, we can find some of them in ShaderToy ( for example https://www.shadertoy.com/view/wlBGDG ) and then rewrite them in Unity's HLSL.

> **Estimation**: 2

### Option 2: Blur
We already have a Gaussian blur shader in the project so we just have to apply it (bake it?) to the target hint texture and show it in the background, we could also create a shader to blend 2 of them together so we can set-up a transition animation between them when changing hints.

> **Estimation**: 3
